### ISSUE TITLE
Deprecation warnings

### ISSUE THREAD

**Issue by milktrader** [JuliaWeb/GnuTLS.jl#23]
````julia
WARNING: deprecated syntax "{a=>b, ...}" at /Users/Administrator/.julia/v0.4/GnuTLS/src/errormap.jl:155.
Use "Dict{Any,Any}(a=>b, ...)" instead.
````

https://github.com/milktrader/Quandl.jl/issues/61

**Comment by sbromberger** [comment]
Fixed via #28

**Comment by milktrader** [comment]
Yay
